src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is working.");
```

Comment:
The message in 'doWork' method should be more informative. For instance, it could indicate what type of machine is working, such as 'Excavator is working', to provide clear context in the logs.